modules/Link.js

```diff
@@ -94,7 +112,7 @@ class Link extends React.Component {
         className={isActive ?
           [ className, activeClassName ].join(' ').trim() : className
         }
-      />
+      >{children}</a>
```

Comment:
Uhm, sorry but I have'nt understand well. Instead of doing `<a {...rest}>{children}</a>` I should do `<a {...rest} />` right? I don't have understand the cloneElement part.